Makefile

```diff
@@ -35,11 +35,19 @@ circle_ci_setup:
 	curl -X POST https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/follow?circle-token=${CIRCLECI_API_KEY}
 
 github_actions_setup:
-	sh scripts/gha-setup.sh
+	sh scripts/gha-setup.sh setup
 
 summary:
 	@echo "zero-deployable-backend:"
 	@echo "- Repository URL: ${REPOSITORY}"
 	@echo "- Deployment Pipeline URL: https://app.circleci.com/pipelines/github/${GITHUB_ORG}/${GITHUB_REPO}"
 	@echo $(shell echo ${ENVIRONMENT} | grep prod > /dev/null && echo "- Production API: ${productionBackendSubdomain}${productionHostRoot}")
 	@echo $(shell echo ${ENVIRONMENT} | grep stage > /dev/null && echo "- Staging API: ${stagingBackendSubdomain}${stagingHostRoot}")
+
+REQUIRED_BINS_GHA := gh
```

Comment:
How about we make the naming a bit more generic so it's easier to copy this around between projects / use as a template for future modules?

Reply:
because this is only required when `CIVendor` is `github-actions`, so each variant's required binaries have to be declared separately (since we cant template on the root makefile)